### ISSUE TITLE
Ornery Sheep in North Gustaberg missing from mob_droplist.sql

### ISSUE THREAD

**Issue by Shiftie-Bassturd** [TabulaRasaXI/TabulaRasa#113]
- [x] I have paid attention to this example and will edit again if need be to not break the formatting, or I will be ignored
- [x] I have searched existing [issues](https://github.com/LandSandBoat/server/issues) to see if the issue has already been opened, and I have checked the commit log to see if the issue has been resolved since my server was last updated
- [x] I have read and understood the [Contributing Guide](https://github.com/LandSandBoat/server/blob/base/CONTRIBUTING.md)

## Branch affected by issue 

`base`

## Steps to reproduce

mob_droplist.sql missing entry for Ornery Sheep in North_Gustaberg (106)

## Expected behavior

North Gustaberg Ornery Sheep should drop the same loot as their counterparts in South_Gustaberg (107) 

-- ZoneID: 107 - Ornery Sheep
INSERT INTO `mob_droplist` VALUES (1958,0,0,1000,4372,210); -- Slice Of Giant Sheep Meat (21.0%)
INSERT INTO `mob_droplist` VALUES (1958,0,0,1000,505,110);  -- Sheepskin (11.0%)
INSERT INTO `mob_droplist` VALUES (1958,0,0,1000,1898,30);  -- Vial Of Fresh Blood (3.0%)
INSERT INTO `mob_droplist` VALUES (1958,0,0,1000,882,20);   -- Sheep Tooth (2.0%)
INSERT INTO `mob_droplist` VALUES (1958,2,0,1000,832,0);    -- Clump Of Sheep Wool (Steal)

![sheepdropswiki](https://user-images.githubusercontent.com/109208330/178706316-6a32cbfa-abcf-4b39-b330-a2eeb5dcd6b4.png)

**Comment by The-Aerec** [comment]
Hi this is an issue but from a coding perspective its slightly wrong in how you have perceived the bug: (Just cause I like telling everyone how the code works cause I think its fun)

How a mob actually drops something is defined in: 
mob_groups.sql

```sql

CREATE TABLE `mob_groups` (
  `groupid` int(10) unsigned NOT NULL,
  `poolid` int(10) unsigned NOT NULL DEFAULT 0,
  `zoneid` smallint(3) unsigned NOT NULL DEFAULT 0,
  `name` varchar(24) DEFAULT NULL,
  `respawntime` int(10) unsigned NOT NULL DEFAULT 0,
  `spawntype` tinyint(3) unsigned NOT NULL DEFAULT 0,
  `dropid` int(10) unsigned NOT NULL DEFAULT 0,
  `HP` mediumint(8) NOT NULL DEFAULT 0,
  `MP` mediumint(8) NOT NULL DEFAULT 0,
  `minLevel` tinyint(2) unsigned NOT NULL DEFAULT 0,
  `maxLevel` tinyint(2) unsigned NOT NULL DEFAULT 0,
  `allegiance` tinyint(2) unsigned NOT NULL DEFAULT 0,
  PRIMARY KEY (`zoneid`,`groupid`)
) ENGINE=MyISAM DEFAULT CHARSET=utf8 AVG_ROW_LENGTH=22;

```

if a mob has that group tag it will drop whats associated in its: dropid slot

For ornery sheep in North Gusta you are right! It has no assigned droplist! (oops)

```sql
INSERT INTO `mob_groups` VALUES (11,3058,106,'Ornery_Sheep',330,0,0,0,0,5,8,0);
```

we can easily just mirror the S_Gustaberg sheep group which is:
```sql
INSERT INTO `mob_groups` VALUES (12,3058,107,'Ornery_Sheep',330,0,1958,0,0,5,8,0);
```

Which leads us to the final edit for N.Gusta sheep mob as:

```sql
INSERT INTO `mob_groups` VALUES (11,3058,106,'Ornery_Sheep',330,0,1958,0,0,5,8,0);
```

This will add the drops to the sheep in north gusta! 

TLDR: 
The code doesn't actually care whats named as the mob in: mob_droplist.sql, it only cares about the ID of that droplist, and that that perticular droplist exists.

We assign the mobs their droplist in mob_groups.sql.

I'll add this to some notes for later for someone to poke at.

**Comment by Shiftie-Bassturd** [comment]
Ah-hah, gotcha! This is very enlightening, it'll be my go to reference moving forward. Thanks for taking the time to write it up Aerec. My limited coding experience was from 2+ decades ago(Pascal in school, debugging/hex editing NES roms and working on text based mmorpgs aka MUDs) all of which I barely remember. Thanks again and cheers!
